src/main/java/JavaQuestionTwo/addUser.java

```diff
@@ -0,0 +1,86 @@
+package JavaQuestionTwo;
+import java.util.*;
+import java.io.*;
+
+
+public class addUser {
```

Comment:
The class name should always start with capital.

Reply:
Done